config/free5gc-ue.yaml

```diff
@@ -4,6 +4,8 @@ supi: 'imsi-208930000000003'
 mcc: '208'
 # Mobile Network Code value of HPLMN (2 or 3 digits)
 mnc: '93'
+# SUCI Protection Scheme : 0 for Null-scheme, 1 for Profile A and 2 for Profile B
+protectionScheme: 0
```

Comment:
homeNetworkPublicKey and homeNetworkPublicKeyId fields are missing. Is that intentional?

Reply:
Free5GC does not support SUCI with Profile A or B. Only Null-Scheme is possible so no key is needed.
May be, in order to be more clear, an empty key could be preferable with a comment explaining why the key is not set.

Reply:
My bad!
Free5GC implements SUCI. I will modify the config file.